### PR TITLE
Added missing redux action types.

### PR DESCRIPTION
**Pull request by Hyperkid123**
I've found some missing constants for redux action types.

@lgalis so there is an action in `PorfolioActions` called `updatePortfolio`. This action is used in some components, but there is no redux listener for `UPDATE_PORTFOLIO` action type. Is it something you are working on right now or is just missing?

### CODE REVIEW

**Review by lgalis — CHANGES_REQUESTED** [review]
ADD_PLATFORM and FETCH_PLATFORM_DATA can be removed. This was a temporary component we used before we switched to the topology service.

**Comment by lgalis** [comment]
@Hyperkid123 - the updatePortfolio action is actually waiting for the updatePortfolio Api to be implemented. I will update that once the api is available.

File: src/redux/ActionTypes.js [hunk 1]
```diff
@@ -17,9 +14,12 @@ export const FETCH_PLATFORM_ITEM =  'FETCH_PLATFORM_ITEM;';
...
 export const FETCH_PORTFOLIOS = 'FETCH_PORTFOLIOS';
+export const FETCH_PORTFOLIO_ITEM = 'FETCH_PORTFOLIO_ITEM';
 export const FETCH_PORTFOLIO_ITEMS = 'FETCH_PORTFOLIO_ITEMS';
 export const FETCH_PORTFOLIO_ITEMS_WITH_PORTFOLIO = 'FETCH_PORTFOLIO_ITEMS_WITH_PORTFOLIO';
 export const FETCH_PORTFOLIO = 'FETCH_PORTFOLIO';
 export const FILTER_PORTFOLIO_ITEMS = 'FILTER_PORTFOLIO_ITEMS';
 export const ADD_PORTFOLIO = 'ADD_PORTFOLIO';
 
+// has no reducer handler
+export const UPDATE_PORTFOLIO = 'UPDATE_PORTFOLIO';
```
Comment (hstastna) [inline src/redux/ActionTypes.js:25]: I added the same line in my PR yesterday :+1:

Comment (Hyperkid123) [reply src/redux/ActionTypes.js:25]: Less conflicts for the future!

**Pull request merged**